src/constants/routing.ts

```diff
@@ -44,7 +44,9 @@ type ChainCurrencyList = {
 }
 
 const WRAPPED_NATIVE_CURRENCIES_ONLY: ChainTokenList = Object.fromEntries(
-  Object.entries(WRAPPED_NATIVE_CURRENCY).map(([key, value]) => [key, [value]])
+  Object.entries(WRAPPED_NATIVE_CURRENCY)
+    .map(([key, value]) => [key, [value]])
+    .filter((token) => !!token)
```

Comment:
You're just using a boolean filter, so be more explicit if typescript allows you to: `.filter(Boolean)`